src/main/scripts/festerize/install.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Determine which Pip 3 executable to use.
+for PIP in pip3 pip; do
+    if ! [ -x "$(command -v ${PIP})" ]; then
+        echo "${PIP} not installed" >&2
+        PIP=
+    else
+        break
+    fi
+done
+
+if [ -z ${PIP} ]; then
+    echo "A version of Pip compatible with Python 3 is required to install Festerize."
```

Comment:
Does the above actually check that `pip` is v3? Are there systems for which `pip` might be pointed to a 2.x version?

Reply:
It could parse `pip -V` output perhaps?

Reply:
Yeah, I think that will work. I verified that it works on a Mac too. Just pushed a new commit.